### ISSUE TITLE
Remember opened component

### ISSUE THREAD

**Issue by ezhlobo** [teamplanes/expobook#3]
Hey, usually (in my opinion) people use live reloading when they change codebase and it's annoying to scroll and open your component every time.

What do you think about remembering opened component and open it again after reloading the app? Like we can store in AsyncStorage state of navigator and restore it after initializing.

**Comment by kirkness** [comment]
Sure makes sense - have you tried using hot-reloading?

**Comment by kirkness** [comment]
Pug in React... pretty interesting! Styled-components shouldn't be a problem with it as no babel config is required.

When you say that you are only using 'its components' - could you elaborate a little? Would be cool to know what your setup looks like and see if we can make things smoother.

Cool PRs are very welcome! Let me know if you'd like a hand.

**Comment by ezhlobo** [comment]
No PR unfortunately. Here is the update: I realized that defining previous component via `initialScreenName` removes 'back to list' button from header. Also I found that we need to create two more components: one for getting previous screen; one for subscribing to changes in navigation. I thought it can be easier :).

My plan for tomorrow or after tomorrow is to check the way via `initialScreenName` (probably we can add 'back to list' button manually) and create those two components.

My plan for near future is to share my use case because it might be interesting for others here.
